src/navigation/DrawerNavigatorItems.js

```diff
@@ -0,0 +1,146 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import {
+  SafeAreaView,
+  StyleSheet,
+  Text,
+  TouchableNativeFeedback,
+  TouchableOpacity,
+  View
+} from 'react-native';
+import { StackActions } from 'react-navigation';
+import { Divider } from 'react-native-elements';
+
+import { colors, device } from '../config';
+
+/**
+ * Component that renders the navigation list in the drawer.
+ *
+ * based on:
+ *   https://github.com/react-navigation/drawer/blob/c5954d744f463e7f1c67941b8eb6914c0101e56c/src/views/DrawerNavigatorItems.tsx
+ */
+const DrawerNavigatorItems = ({
+  items,
+  activeItemKey,
+  activeTintColor,
+  activeBackgroundColor,
+  inactiveTintColor,
+  inactiveBackgroundColor,
+  getLabel,
+  itemsContainerStyle,
+  itemStyle,
+  labelStyle,
+  activeLabelStyle,
+  inactiveLabelStyle,
+  drawerPosition,
+  navigation
+}) => {
+  const Touchable = device.platform === 'ios' ? TouchableOpacity : TouchableNativeFeedback;
+
+  /**
+   * based on:
+   *   https://github.com/react-navigation/drawer/blob/c5954d744f463e7f1c67941b8eb6914c0101e56c/src/views/DrawerSidebar.tsx#L67
```

Comment:
this link doesn't lead to a specific page , but the the google search react-navigation/drawer.

Reply:
for me this link is working fine: https://github.com/react-navigation/drawer/blob/c5954d744f463e7f1c67941b8eb6914c0101e56c/src/views/DrawerSidebar.tsx#L67